### PR TITLE
Banner stats: minor html

### PR DESCRIPTION
**Pull request by torvista**

### CODE REVIEW

File: admin/banner_statistics.php [hunk 1]
```diff
@@ -178,7 +178,7 @@
           });
 
           function showTooltip(x, y, contents) {
-              $('<div id="tooltip">' + contents + '</div>').css({
+              $('<div id="tooltip">' + contents + '<\/div>').css({
```
Comment (scottcwilson) [inline admin/banner_statistics.php:181]: Are you sure? Why?

File: admin/banner_statistics.php [hunk 2]
```diff
@@ -314,7 +314,7 @@ function showTooltip(x, y, contents) {
         </script>
       </div>
 
-<?php if (FALSE) { ?>
+<?php if (FALSE) { //https://github.com/zencart/zencart/issues/5016 ?>
```
Comment (scottcwilson) [inline admin/banner_statistics.php:317]: Let's just get rid of this.  We have source control if someone wants to resurrect it.

File: admin/banner_statistics.php (hunk 1 above)
Comment (torvista) [reply admin/banner_statistics.php:181]: Surprise, surprise:

image

Comment (scottcwilson) [reply admin/banner_statistics.php:181]: thanks for clarifying.

**Pull request merged**